### PR TITLE
app-crypt/adcli: bump version to 0.9.2

### PR DESCRIPTION
**Pull request by henning-schild**
Signed-off-by: Henning Schild <henning@hennsch.de>

### CODE REVIEW

File: app-crypt/adcli/adcli-0.9.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+BDEPEND="
+	doc? (
+		app-text/docbook-xml-dtd:4.3
+		app-text/xmlto
+		dev-libs/libxslt
+	)"
+
+src_prepare() {
+	default
+	eautoreconf
```
Comment (juippis) [inline app-crypt/adcli/adcli-0.9.2.ebuild:30]: Is this needed?

Comment (henning-schild) [reply app-crypt/adcli/adcli-0.9.2.ebuild:30]: Yes, the upstream code-base and the release tarballs do not contain things like configure-script and Makefile. Those need to be generated.

**Comment by henning-schild** [comment]
https://bugs.gentoo.org/880087

**Review by juippis — APPROVED** [review]
Sorry for missing an update earlier on this PR.

**Pull request closed without merging**